### DISCUSSION TITLE
I want to get the 'lastFailureReason' from a 'RecordContext', how can I do that?

### DISCUSSION THREAD

**Discussion by jvissers** [confluentinc/parallel-consumer#720] [Q&A] [answered]
Is there a way that lets met get to the 'last failure reason' which is contained by the `WorkContainer`   inside of `RecordContext`?

**Comment by rkolesnev** [comment]
Can you elaborate please? where are you trying to access it from?
I think accessor for it may need to be added to RecordContext - if you can submit a PR to do that - should be quick.

**Reply by jvissers** [reply]
For our UI we want to be able to have the technical reason why a certain event after n failed attempts, still failed to be consumed - and right now that information from `RecordContext` is something we cannot get to. 

https://github.com/confluentinc/parallel-consumer/pull/726

**Comment by rkolesnev** [comment, marked as answer]
Implemented by @jvissers - thanks - and merged in #727
